activemq-artemis/templates/service.yaml

```diff
@@ -6,7 +6,7 @@ kind: Service
 metadata:
   annotations:
     # Make sure DNS is resolvable during initialization.
-    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "false"
```

Comment:
perhaps the comment above should be changed

Reply:
Agreed. Will update it to reflect the intent.

Reply:
Updated in latest commit